pyirf/io/gadf.py

```diff
@@ -325,3 +304,24 @@ def create_rad_max_hdu(
     _add_header_cards(header, **header_cards)
 
     return BinTableHDU(rad_max_table, header=header, name=extname)
+
+
+def compare_irf_cuts(cuts):
```

Comment:
This raises an error if the cuts are not equal. With the current name, I would expect it to return `True` or `False`, or I would change the name to `check_cuts_are_equal`. I would probably go with returning `False`.

This function would also be better placed in `pyirf/cuts.py`, has nothing to do with IO. 


So, I would put this in `pyirf/cuts.py`, and replace the `raise ValueError(...)` with `return False`.

Reply:
moved and changed to return False